modules/update-service-overview.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='update-service-overview-{context}']
+== The {product-title} update service
+
+The {product-title} update service is the hosted service that provides over the air updates to both 
+{product-title} and RHCOS. It provides a graph, or diagram that contain
+_vertices_ and the _edges_ that connect them, of component Operators. The edges
+in the graph show which versions you can safely upgrade to. The cluster version
+Operator checks with the {product-title} update service and determines valid upgrades and upgrade paths
+based on current component versions and information in the graph. If you
+configure it to do so, the {product-title} update service sends the release artifacts that it needs to
+perform the upgrade to your image registry, and the cluster version Operator
+upgrades your cluster. By accepting automatic updates, you can automatically
+keep your cluster up to date with the most recent compatible components.
+
+To allow the {product-title} update service to provide only compatible updates, a release verification
+pipeline exists to drive automation. Each release artifact is verified for
+compatibility with supported cloud platforms and system architectures as well
+as other component packages. After the pipeline confirms the suitability of a 
+release, the {product-title} update service can apply the update to your cluster or notify you that it
+is available.
+
+The interaction between the registry and the {product-title} update service is different during
+bootstrap and continuous update modes. When you bootstrap the initial
+infrastructure, the cluster version Operator finds 
+the fully qualified image name for the shortname of the images that it needs to 
+apply to the server during installation. It looks at the image stream that it needs
+to apply and renders it to disk. It calls bootkube and waits for a temporary minimal control
+plane to come up and load the cluster version Operator.
+
+During continuous update mode, two controllers run. One continuously updates
+the payload manifests, applies them to the cluster, and outputs the status of
+the controlled rollout of the Operators, whether they are available, upgrading,
+or failed. The second controller constantly checks with to Cincinnati to
```

Comment:
This is your first mention of Cincinnati by name.  If we want to name it, we should probably name it early in this page.  If we want to leave it generic, we should probably replace Cincinnati with "the {product-title} update service".

And maybe "constantly checks" -> "polls"?  "constantly checks" sounds like a hot spin-lock, which isn't very civilized ;).